tests/unit/moduleapi/defrag.tcl

```diff
@@ -63,6 +63,7 @@ start_server {tags {"modules"} overrides {{save ""}}} {
             assert {[getInfoProperty $info defragtest_global_strings_attempts] > 0}
             assert {[getInfoProperty $info defragtest_global_dicts_attempts] > 0}
             assert {[getInfoProperty $info defragtest_global_dicts_defragged] > 0}
+            assert {[getInfoProperty $info defragtest_global_dicts_items_defragged] > 0}
```

Comment:
@oranagra Add a new metric to check the dicts items being defragged correctly.

Reply:
i just hope the test is not flaky (i.e. that normally when we run it we don't see a very low number here, that can become 0 some day

Reply:
It is safe. I have run more than 10 full CIs for modules since last night without any failure.
Since we have already deleted half of the 10 elements in a dictionary in the preview PR, fragmentation is bound to occur.